### ISSUE TITLE
textDocument/references disabled

### ISSUE THREAD

**Issue by saibing** [autozimu/LanguageClient-neovim#14]
textDocument/references is disabled, it is report:

```sh
No selection UI found. Consider install fzf or denite.vim.
```

Maybe the follow code not initialize variable: selectionUI

```python
    @neovim.function('LanguageClient_textDocument_references')
    def textDocument_references(self, args: List) -> None:
        if not self.alive(warn=True):
            return
        logger.info("Begin textDocument/references")

        uri, line, character, sync, cb = self.getArgs(
                args, ["uri", "line", "character", "sync", "cb"])
        if not sync and not cb:
            cb = partial(
                    self.handleTextDocumentReferencesResponse,
                    self.getSelectionUI())
```

**Comment by autozimu** [comment]
No, it means you should have fzf or denite to the feature, as the error message says.

When you have fzf installed, it should allow you to selected from the menu.

Or if you denite installed, use `denite references` to see the menu.

**Comment by autozimu** [comment]
Sorry, my mistake.

Thanks for reporting.

**Comment by autozimu** [comment]
Should been fixed by 0d38097353bf497de7165462fccc6697f39a3a55
